src/trackers/AVISTAZ_NETWORK.py

```
@@ -454,13 +454,36 @@ async def get_screenshots(self, meta):
+            with open(path, 'rb') as f:
+                image_bytes = f.read()
+            return await self.img_host(meta, self.tracker, image_bytes, path.name)
+
+        async def upload_remote_file(url):
+            try:
+                response = await self.session.get(url)
+                response.raise_for_status()
+                image_bytes = response.content
+                filename = os.path.basename(urlparse(url).path) or 'screenshot.png'
+                return await self.img_host(meta, self.tracker, image_bytes, filename)
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)